modules/Library/library_manage_catalog.php

```diff
@@ -287,7 +287,7 @@
                 }
                 if ($resultSpace->rowCount() == 1) {
                     $rowSpace = $resultSpace->fetch();
-                    echo $rowSpace['name'].'<br/>';
+                    echo __($rowSpace['name']).'<br/>';
```

Comment:
I am not sure it makes sense to wrap this one, as it is a user-entered value, and so is unlikely to be in the string translation list, but is likely to be in the school's language already. Same with __($rowSlots['nameShort']) in  modules/Activities/report_attendanceExport.php I believe. I could be wrong though...

Reply:
This really does not make sense here, but in modules/Activities/report_attendanceExport.php the value of  $rowSlots['nameShort'] is not user-entered.